ansible/tasks/recreateViews.yml

```diff
@@ -3,20 +3,23 @@
 
 - include: db/recreateDoc.yml
   vars:
-    dbName: "{{ db.whisk.actions }}"
-    doc: "{{ lookup('file', '{{ openwhisk_home }}/ansible/files/whisks_design_document_for_actions_db.json') }}"
-- include: db/recreateDoc.yml
-  vars:
-    dbName: "{{ db.whisk.actions }}"
-    doc: "{{ lookup('file', '{{ openwhisk_home }}/ansible/files/filter_design_document.json') }}"
+    dbName: "{{ item.dbName }}"
+    doc: "{{ lookup('file', '{{ item.doc }}') }}"
+  with_items:
+    - { dbName: '{{ db.whisk.actions }}', doc: '{{ openwhisk_home }}/ansible/files/whisks_design_document_for_actions_db.json' }
+    - { dbName: '{{ db.whisk.actions }}', doc: '{{ openwhisk_home }}/ansible/files/filter_design_document.json' }
 
 - include: db/recreateDoc.yml
   vars:
-    dbName: "{{ db.whisk.activations }}"
-    doc: "{{ lookup('file', '{{ openwhisk_home }}/ansible/files/whisks_design_document_for_activations_db.json') }}"
+    dbName: "{{ item.dbName }}"
+    doc: "{{ lookup('file', '{{ item.doc }}') }}"
+  with_items:
+    - { dbName: '{{ db.whisk.activations }}', doc: '{{ openwhisk_home }}/ansible/files/whisks_design_document_for_activations_db.json' }
+    - { dbName: '{{ db.whisk.activations }}', doc: '{{ openwhisk_home }}/ansible/files/filter_design_document.json' }
   when: db.whisk.activations != db.whisk.actions
+
+
 - include: db/recreateDoc.yml
   vars:
     dbName: "{{ db.whisk.activations }}"
-    doc: "{{ lookup('file', '{{ openwhisk_home }}/ansible/files/filter_design_document.json') }}"
-  when: db.whisk.activations != db.whisk.actions
+    doc: "{{ lookup('file', '{{ openwhisk_home }}/ansible/files/activations_design_document.json') }}"
```

Comment:
Can we move this command in the loop above.
And we should create the document in both databases.